instamatic/processing/ImgConversion.py

```diff
@@ -647,6 +647,51 @@ def write_pets_inp(self, path: str, tiff_path: str = 'tiff') -> None:
                 angle = self.start_angle + sign * self.osc_angle * i
                 print(f'{tiff_path}/{fn} {angle:10.4f} 0.00', file=f)
             print('endimagelist', file=f)
+    
+    def write_pets2_inp(self, path: str, tiff_path: str = 'tiff') -> None:
+        """Write PETS 2 input file `pets.pts2` in directory `path`"""
+        path.mkdir(exist_ok=True, parents=True)
+
+        if self.start_angle > self.end_angle:
+            sign = -1
+        else:
+            sign = 1
+
+        omega = np.degrees(self.rotation_axis)
+
+        with open(path / 'pets.pts2', 'w') as f:
+            date = str(time.ctime())
+            print(f'# PETS 2 input file for Rotation Electron Diffraction generated by `instamatic`', file=f)
+            print(f'# {date}', file=f)
+            print(f'# For definitions of input parameters, see:', file=f)
+            print(f'# http://pets.fzu.cz/ ', file=f)
+            print(f'', file=f)
+            print(f'geometry continuous', file=f)
+            print(f'lambda {self.wavelength}', file=f)
+            print(f'Aperpixel {self.pixelsize}', file=f)
+            print(f'phi {float(self.osc_angle)/2}', file=f)
+            print(f'omega {omega}', file=f)
+            print(f'bin 1', file=f)
+            print(f'reflectionsize 15', file=f)
+            print(f'noiseparameters 25 10', file=f)
+            print(f'i/sigma    5.00   10.00', file=f)
+            print(f'', file=f)
```

Comment:
fair enough :) this was just a copy/paste and I just left it in, in case I wanted to have some of these values as variables. But as you suggested, it's probably better to remove the 'f'.